packages/theme-chalk/src/common/var.scss

```diff
@@ -1037,6 +1037,7 @@ $menu: map.merge(
     'hover-bg-color': getCssVar('color-primary-light-9'),
     'item-height': 56px,
     'sub-item-height': calc(#{getCssVar('menu-item-height')} - 6px),
+    'horizontal-height': 60px,
```

Comment:
Why is it 60 here? Maybe it can be consistent with `item-height`?

Reply:
`item-height` 56px + blue border-bottom 2px + memu border 1px = 59px

I think it's a multiple of four in height that's more canonical and easier to remember. 60px is the closest number.

Change this height dynamically, and the content inside is always centered.

Reply:
Ok.